frontend/apps/cloudstorage/dropbox.lua

```diff
@@ -10,9 +10,280 @@ local util = require("util")
 local Screen = require("device").screen
 local T = require("ffi/util").template
 local _ = require("gettext")
+local lfs = require("libs/libkoreader-lfs")
 
 local DropBox = {}
 
+function DropBox:showFilesAndFolders(url, password)
+    return DropBoxApi:showFilesAndFoldersV2(url, password)
+end
+
+function DropBox:getRemoteFilesMap(url,password,treePath)
```

Comment:
Our coding style says that camelCase is a function for quick identification.
```suggestion
function DropBox:getRemoteFilesMap(url, password, tree_path)
```